src/Tiqr/Legacy/TiqrUserRepository.php

```diff
@@ -39,6 +39,7 @@ public function __construct(private Tiqr_UserStorage_Interface $userStorage, pri
     }
 
     /**
+     * @throws UserNotExistsException
```

Comment:
Are these `@throws` additions required? Or a left over of you previous solution.

Reply:
An addition from phpstorm, who suggested on another function (not touched further btw)